src/DeepNgRoot/hook.init.js

```diff
@@ -1,22 +1,25 @@
 /**
  * Created by AlexanderC on 10/6/15.
  */
- 
+
 'use strict';
 
 var exports = module.exports = function(callback) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["exports" is defined but never used](https://www.codacy.com/app/MitocGroup/deep-microservices-root-angularjs/file/1796871863/issues/source?bid=2928423&fileBranchId=3072037#l7)